### PR TITLE
i18n(es): Translated `external-links.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Translated `external-links.mdx`

### CODE REVIEW

File: src/content/docs/es/recipes/external-links.mdx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: AAgregar iconos a los enlaces externos
```
Comment (dreyfus92) [inline src/content/docs/es/recipes/external-links.mdx:2]:
```suggestion
title: Agregar iconos a los enlaces externos
```

**Review by dreyfus92 — COMMENTED** [review]
Buen trabajo @Waxer59, te dejo una sugerencia 😁

**Comment by Waxer59** [comment]
Gracias por la revisión 🤠

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

**Review by Yan-Thomas — COMMENTED** [review]
LGTM! Great job peeps 🚀

**Pull request merged**